newcomers-tutorial.md

```diff
@@ -179,14 +176,13 @@ All is fine.
 
 ### Rebuilding
 
-While working on GHC, you will need to rebuild often; however, most of the
-time, it is possible to avoid a full, slow build. Here's a few things you can
-do to speed things up:
+While working on GHC you will need to rebuild often. Most of the time, it is
```

Comment:
I'd keep the comma due to inverted sentence structure.

I'd also keep one of "however", "though", or "but" in there, to keep the "this is true, but you can do that" logic in there. What I'm trying to say here is not "We would like to remind you to rebuild often during development, and you can speed this up by doing the following", but rather "Unfortunately, frequent rebuilds are inevitable during development, but you can alleviate some of the pain by speeding things up, and here's how". So the "but" notion is kind of important here.

Reply:
None of those semantics are lost by the rework, but it *does* result in a sentence that flow better (IMO). I'll see if I can meet us both halfway somehow.

Reply:
Okay I reflowed it and reduce it. Hopefully you like that more.